docs/rst/tutorials/cloud_tutorial.rst

```diff
@@ -0,0 +1,166 @@
+* A `Local Discovery Server <https://eprosima-dds-router.readthedocs.io/en/latest/rst/user_manual/participants/local_discovery_server.html>`_ (local DS) that propagates them to a ROS 2 listener node hosted in a different *K8s* pod.
+
+The choice of a Local Discovery Server instead of a Simple Participant to communicate with the listener has to do with the difficulty of enabling multicast routing in cloud environments.
+
+The described scheme is represented in the following figure:
+
+.. figure:: /rst/figures/cloud_tutorial/vulcanexus_cloud.png
+
+In addition to the two mentioned deployments, two *K8s* `services <https://kubernetes.io/docs/concepts/services-networking/service/>`_ are required in order to direct dataflow to each of the pods.
+A LoadBalancer will forward messages reaching the cluster to the WAN participant of the cloud router, and a ClusterIP service will be in charge of delivering messages from the local DS to the listener pod.
+Following are the settings needed to launch these services in *K8s*:
+
+.. literalinclude:: ../../resources/tutorials/cloud_tutorial/load-balancer-service.yaml
+    :language: yaml
+
+.. literalinclude:: ../../resources/tutorials/cloud_tutorial/local-service.yaml
+    :language: yaml
+
+.. note::
+
+    An `Ingress <https://kubernetes.io/docs/concepts/services-networking/ingress/>`_ needs to be configured for the LoadBalancer service to make it externally-reachable.
+    In this example we consider the assigned public IP address to be ``2.2.2.2``.
+
+The configuration file used for the cloud router will be provided by setting up a `ConfigMap <https://kubernetes.io/docs/concepts/configuration/configmap/>`_:
+
+.. literalinclude:: ../../resources/tutorials/cloud_tutorial/ConfigMap.yaml
+    :language: yaml
+
+Following is a representation of the overall *K8s* cluster configuration:
+
+.. figure:: /rst/figures/cloud_tutorial/vulcanexus_k8s.png
+
+
+DDS-Router deployment
+"""""""""""""""""""""
+
+The cloud router is launched from within a Vulcanexus Docker image, which uses as configuration file the one hosted in
+the previously set up ConfigMap. Assuming the name of the generated Docker image is ``ubuntu-vulcanexus-cloud:galactic``, the
+cloud router will then be deployed with the following settings:
+
+.. literalinclude:: ../../resources/tutorials/cloud_tutorial/ddsrouter.yaml
+    :language: yaml
+
+
+Listener deployment
+"""""""""""""""""""
+
+Since ROS 2 demo nodes package is not installed by default in *Vulcanexus Cloud*, a new Docker image adding in this functionality should be generated.
+Also, the IP address and port of the local Discovery Server must be specified, so a custom entrypoint is also provided.
+
+Copy the following snippet and save it to the current directory as ``Dockerfile``:
```

Comment:
Please change ROS_DISTRO to VULCANEXUS_DISTRO everywhere.

Reply:
I have changes it in most places. However, I have kept it in the `Dockerfile_cloud` file when installing the `demo_nodes_cpp` package. I am not sure if in this case we should also change it as this package is not forked in the Vulcanexus distribution and it is taken directly from the ROS distribution. Even though currently `ROS_DISTRO` and `VULCANEXUS_DISTRO` match, it may change in the future. But if this was the case, I suppose Vulcanexus will have a package with its distro... IDK. @EduPonz, what do you think?

Reply:
Same as my comment above, I would directly write `galactic` and avoid the fuss.

Reply:
I think that actually there it could say `VULCANEXUS_DISTRO` for sourcing and `ROS_DISTRO` for installing, but I have no problem with it saying galactic directly, as the `FROM ubuntu-vulcanexus-cloud:galactic` makes the image not a template anyways

Reply:
Then if there is no one against it we use `galactic` and then we avoid the issue. I have made a last commit where the generalization using environment variables has been removed and `galactic` is directly used.

Reply:
:+1: